src/features/defi/providers/idle/components/IdleManager/Claim/ClaimCommon.ts

```diff
@@ -20,33 +20,16 @@ type IdleClaimValues = EstimatedGas & {
   usedGasFee: string
 }
 
-// Redux only stores things that are serializable. Class methods are removed when put in state.
-type SerializableOpportunity = Omit<
-  IdleOpportunity,
-  | 'allowance'
-  | 'prepareApprove'
-  | 'prepareDeposit'
-  | 'prepareWithdrawal'
-  | 'prepareClaimTokens'
-  | 'signAndBroadcast'
-  | 'getClaimableTokens'
->
-
 export type IdleClaimState = {
-  opportunity: SerializableOpportunity | undefined
-  userAddress: string | undefined
   approve: EstimatedGas
   claim: IdleClaimValues
-  claimableTokens: ClaimableToken[]
   loading: boolean
   txid: string | undefined
 }
 
 export enum IdleClaimActionType {
   SET_OPPORTUNITY = 'SET_OPPORTUNITY',
-  SET_USER_ADDRESS = 'SET_USER_ADDRESS',
   SET_CLAIM = 'SET_CLAIM',
-  SET_CLAIMABLE_TOKENS = 'SET_CLAIMABLE_TOKENS',
```

Comment:
Useless as an action now, we can get this from the populated opportunities slice